Druid-DispatchTable/PCGMessageNode.extension.st

```diff
@@ -0,0 +1,16 @@
+Extension { #name : #PCGMessageNode }
+
+{ #category : #'*Druid-DispatchTable' }
+PCGMessageNode >> arguments: anObject [
+
+	arguments := anObject
+]
```

Comment:
This is used by `PCGMessageNode class>>receiver:selector:argument:` which I added as extension method because Object>>asOrderedCollection was deprecated in Pharo 11. And this is used by #buildPrimitiveTableInitialValueNode. 

We could change it but I think it's nice to have argument: for passing a single PCG object, and arguments: when passing a Collection.

Reply:
I already fixed the problem with `asOrderedCollection` by using collections where they are needed :)

Reply:
I don't understand. Where it is fixed? 
Does it mean I should change the users of `PCGMessageNode class>>receiver:selector:argument: ` ?